### PR TITLE
Always add python-pip and python3-pip to apt install list

### PR DESCRIPTION
**Pull request by zmichaels11**
`colcon-bundle` has a bug where passing `--pip-requirements` or `--pip3-requirements` without a python package defined in `setup.py` will cause pip (or pip3) to not be installed.

This workaround always installs `python-pip` and `python3-pip` in the bundle.

### How to test manually
* Delete `integration/test_workspace/src/test-py-module`
* Install dependencies
* Source ROS 1
* Run `colcon build && colcon bundle --pip-requirements py27_requirements.txt --pip-requirements py3_requirements.txt`

Signed-off-by: Zachary Michaels <zmichaels11@gmail.com>

### CODE REVIEW

**Comment by zmichaels11** [comment]
This PR is purposefully kept in draft because it might break workflows where Python3 is being installed through an external means (like a PPA).

File: colcon_bundle/installer/apt.py [hunk 1]
```diff
@@ -154,6 +154,10 @@ def setup(self):  # noqa: D102
                                'you set your keys correctly?')
         self._cache.open()
 
+        # Workaround for pip-requirements not installing python-pip
+        self.add_to_install_list('python-pip')
```
Comment (dirk-thomas) [inline colcon_bundle/installer/apt.py:158]: This package doesn't exist on Ubuntu Focal anymore.

Comment (zmichaels11) [reply colcon_bundle/installer/apt.py:158]: thanks, I'll update the commit

Comment (prajakta-gokhale) [reply colcon_bundle/installer/apt.py]: Removed

**Comment by prajakta-gokhale** [comment]
FYI @jerwallace

**Comment by thomas-moulard** [comment]
Make sure to evaluate the impact on performance before merging this. It's going to make each bundle bigger. Is the additional weight acceptable?

File: colcon_bundle/installer/apt.py [hunk 2]
```diff
@@ -154,6 +154,9 @@ def setup(self):  # noqa: D102
                                'you set your keys correctly?')
         self._cache.open()
 
+        # Workaround for pip-requirements not installing python-pip
```
Comment (thomas-moulard) [inline colcon_bundle/installer/apt.py:157]: TODO(colcon/colcon-bundle): stop bundling PIP3 and PIP all the time

**Comment by piraka9011** [comment]
Created a dummy package with a single launch file, `CMakeLists.txt` and `package.xml`.
Dummy `requirements.txt`used:
```
flake8==3.8.1
```

Results of this change to the size of the bundle `output.tar`:

| Branch                   | Size | Command                                              |
|--------------------------|------|------------------------------------------------------|
| colcon-bundle:master     | 323M | `colcon bundle`                                      |
| colcon-bundle:master     | 325M | `colcon bundle --pip3-requirements requirements.txt` |
| colcon-bundle:workaround | 325M | `colcon bundle`                                      |
| colcon-bundle:workaround | 327M | `colcon bundle --pip3-requirements requirements.txt` |

cc: @thomas-moulard

**Pull request merged**